README.md

```diff
@@ -62,3 +62,29 @@ If you're adding Vundle to a built-up vimrc, just make sure all these calls
 3. Open vim and run `:PluginInstall`.
 
 To update, open vim and run `:PluginInstall!` (notice the bang!)
+
+
+Known Issues
+------------
+
+We use `setfiletype` upon autodetect, which does not overrides filetype once it
```

Comment:
did you mean `which overrides previous values` instead of `which does not overrides`?